Apply-For-Me-Api/src/main/java/com/hydraulic/applyforme/controller/ProfessionalController.java

```diff
@@ -27,4 +31,9 @@ public List<Professional> findAll(@RequestParam(required = false, defaultValue =
     public Professional findOne(@PathVariable(name ="id") Long id) {
         return service.findOne(id);
     }
+
+    @PutMapping("/update/{id}")
+    public Professional update(@Validated @RequestBody ProfessionalDto body, @PathVariable(name ="id") Long id) {
```

Comment:
There is now a working logic for authenticating user, so instead of passing the id as a pathvariable in the url.
What to do:
1. remove the {id} 
2. use the  CurrentUserUtil.getCurrentUser() to get the authenticated 
3. use the instance generated in step 2 to get the id of the user 
4. pass the id as a parameter to you service and things work well.

basically there is no need to use id on the url